### PR TITLE
new icon: photonengine (original,plain)

### LINKED ISSUE #1469: [ICON REQUEST]: Photonengine
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Photon is a Multiplier Gaming Engine Backend SDK

### Links and sources

https://www.photonengine.com
This is a multiplayer gaming engine service provided to build multiplayer games in unity

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by yeyeto2788**
## Double check these details before you open a PR

- [X] PR does not match another non-stale PR currently opened
- [X] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [X] PR's base is the `develop` branch.
- [X] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [X] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [X] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1469**

### CODE REVIEW

**Comment by Snailedlt** [comment]
I made some changes:
- Added plain icon
- Edited `font` and `alias` in devicon.json
- Moved the devicon.json entry up, so that it's alphabetically sorted

**Review by Snailedlt — DISMISSED** [review]
LGTM! ✔️
Thank you for your contribution! 🙇

File: devicon.json [hunk 1]
```diff
@@ -8656,6 +8656,31 @@
...
+          "photon"
+        ],
+        "tags": [
+            "game",
+            "game-engine"
+        ],
+        "versions": {
+            "svg": ["original"],
+            "font": ["plain"]
+        },
```
Comment (lunatic-fox) [inline devicon.json:8671]:
```suggestion
        "versions": {
            "svg": [
                "original"
            ],
            "font": [
                "plain"
            ]
        },
```

**Review by Snailedlt — APPROVED** [review]

File: devicon.json [hunk 2]
```diff
@@ -8656,6 +8656,35 @@
...
+            "game-engine"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
```
Comment (lunatic-fox) [inline devicon.json:8675]:
```suggestion
        "versions": {
            "svg": [
                "original",
                "plain"
            ],
            "font": [
                "plain"
            ]
        },
```
Adds `plain` version in `svg[]`.

**Review by lunatic-fox — CHANGES_REQUESTED** [review]

**Comment by lunatic-fox** [comment]
`plain` version added to `versions.svg[]`

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Great contribution! 🚀

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**